spec/features/h3_object_creation_spec.rb

```diff
@@ -89,9 +89,11 @@
     expect(page).to have_text('Welcome to Argo!')
 
     visit "#{Settings.argo_url}/view/#{work_druid}"
-    expect(page).to have_text('v1 Accessioned')
+    reload_page_until_timeout!(text: 'v1 Accessioned')
```

Comment:
I was noticing that Argo was taking a bit of time to catch up to H3 in QA saying it was deposited, this keeps the test from stopping immediately and waits for accessioning to be fully complete according to the Argo index